Sources/App/Models/Package.swift

```diff
@@ -100,13 +100,14 @@ extension Package {
     static func findPreRelease(_ versions: [Version], after release: Reference?) -> Version? {
         versions
             .filter { $0.reference?.semVer != nil }
-            .sorted { $0.reference!.semVer! > $1.reference!.semVer! }
+            .filter { $0.commitDate != nil }
+            .sorted { $0.commitDate! > $1.commitDate! }
```

Comment:
Only doing this for `findPreRelease`, because `findRelease` is correctly sorting by semantic version `(Int, Int, Int)`, which is arguably a stronger criterion than commit date.

Where this would fall down is if someone did this:

|     |
|-b2  |
|     |
|     |-b3
|     |
|-b1 /
|   /
|  /
| /
|/

🤷‍♂️

Reply:
Yea I think that situation is going to be very rare. I can't think of a good reason anyone would do this (which doesn't mean people won't do it, but it does mean we're allowed to care a little less about it 😂)